### DISCUSSION TITLE
[Feature Request] Remove/Block Livestreams from Shorts

### DISCUSSION THREAD

**Discussion by Zeroshiki-0** [ReVanced/revanced-discussions#1041] [General]
Now that YouTube has added Livestreams to Shorts, they're ruining my recommendations, in and outside of Shorts, and are getting really annoying. Even after scrolling past them immediately or clicking "Not Interested", they add the entire stream to your history and if you don't remove it right away, the algorithm thinks you want similar content pushed to you. Terrible decision imo. Is it possible to remove or block them entirely ?

**Comment by LisoUseInAIKyrios** [comment]
And many of these live stream Shorts pollute the watch history because they later show up as regular video (because they're not actually Shorts, but vertical orientation live stream regular videos). 

I don't know what YT is thinking with their recommendations, because the videos are usually awful quality with less than a few hundred views from new and unknown channels.

Currently there is no way to block or skip them.

I agree it would be great if they could be somehow avoided.

**Reply by Zeroshiki-0** [reply]
Yes, it's getting increasingly frustrating. I see why they're doing it, it gives more engagement to smaller channels and pushes them further out, but having these smaller channels pop up is exactly why they keep recommending random videos from random small channels if you don't wipe the livestreams from your history. And even then, that doesn't seem to be a fix.

**Reply by oSumAtrIX** [reply]
In my experience, I have only ever seen a single Shorts stream. To filter shorts, the field that returns the list of shorts has to be modified, relatively early too, since we have hooks around what video is playing for example

**Comment by chanceofdraw** [comment]
Have you since found a solution to this problem? I am having the same issue with lives in the shorts.

**Reply by Zeroshiki-0** [reply]
Nope, I wish. I'm still Googling it every few months to see if someone's finally fixed it.

**Comment by PrdpChinna** [comment]
It's Sep 2025, still there is no option to remove live content from shorts feed.

**Reply by LisoUseInAIKyrios** [reply]
You are invited to spend your time reverse engineering a way to hide these videos from the Shorts feed.

**Comment by mvfsullivan** [comment]
4 years later..... Any solution? :(
